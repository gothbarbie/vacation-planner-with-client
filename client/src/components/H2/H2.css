.h2 {
  font-size: 23px;
  font-weight: 300;
  padding: 1rem;
  color: #333333;
}
