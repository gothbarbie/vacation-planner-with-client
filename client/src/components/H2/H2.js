// @flow

import * as React from 'react'
import classnames from 'classnames'
import './H2.css'

type props = {
  children: React.ChildrenArray<any>,
  light?: boolean,
  dark?: boolean,
}

const H2 = ({ children, light, dark }: props) => (
  <h2
    className={classnames('h2', {
      'h2--light': light,
      'h2--dark': dark,
    })}>
    {children}
  </h2>
)

export default H2
