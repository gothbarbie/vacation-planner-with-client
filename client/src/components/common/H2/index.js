// @flow

import * as React from 'react'
import classnames from 'classnames'
import './H2.css'

type props = {
  children: React.ChildrenArray<any>,
  light?: boolean,
  dark?: boolean,
}

export default ({ children, light, dark }: props) => (
  <h2
    className={classnames('h2', {
      'h2--light': light,
      'h2--dark': dark,
    })}>
    {children}
  </h2>
)
