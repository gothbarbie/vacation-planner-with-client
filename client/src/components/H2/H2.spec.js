import React from 'react'
import { shallow } from 'enzyme'

import H2 from './H2'

describe('components/H2', () => {
  const h2 = shallow(<H2 />)

  it('renders', () => {
    expect(h2).toMatchSnapshot()
  })
})
