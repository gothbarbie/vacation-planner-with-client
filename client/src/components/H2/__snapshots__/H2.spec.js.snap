// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/H2 renders 1`] = `
<h2
  className="h2"
/>
`;
